kombu/transport/SQS.py

```diff
@@ -153,7 +153,9 @@ def _reset_cycle(self):
     def entity_name(self, name, table=CHARS_REPLACE_TABLE):
         """Format AMQP queue name into a legal SQS queue name."""
         if name.endswith('.fifo'):
-            partial = name.rstrip('.fifo')
+            # SQS allows no dots in queue names except FIFO queues.
+            # For FIFO queues only one dot is allowed in '.fifo' suffix
+            partial = name.split('.')[:1][0]
```

Comment:
Nice finding! A suggestion here, I think this is equivalent and perhaps simpler:

```python
partial = name.split('.')[0]
```

Reply:
makes sense, but I initially thought SQS may have multiple '.' in queue name..

Reply:
you are correct... I over complicated it for no reason

Reply:
Is the dot character valid in queue names? If that is the case and perhaps accounts better for all possible cases:

```python
>>> 'a.b.c.fifo'.rsplit('.', 1)
['a.b.c', 'fifo']
```